### PR TITLE
Add google analytics to LIFF

### PR DESCRIPTION
**Pull request by MrOrz**
## Feature
- Google analytics setup for LIFF; use the same Google analytics tracking ID as chatbot event handlers
- sends pageview on page change
- sends user timing on LIFF load
- record redirect count as redirect event

## Bugfix
Fix the bug that article list LIFF showing all replies as unread replies.
After fix: 
image

## Pageview
image

## Event
image

## `utm_source` & `utm_medium` in LIFF URL also works
image

### CODE REVIEW

**Comment by MrOrz** [comment]
I forgot to note the new GA event / pageview in README.

**Comment by MrOrz** [comment]
Confirmed that the `LIFF` / `chooseArticle` is working. Ready to review now.
image

File: README.md [hunk 1]
```diff
@@ -271,3 +273,20 @@ Sent event format: `Event category` / `Event action` / `Event label`
 
 7. User updates their reason of reply request
   - `Article` / `ProvidingReason` / `<articleId>`
+  - When the LIFF opens, page view for page `/reason` is also sent.
+
+8. User opens article list
+  - Page view for page `/articles` is sent
+  - If opened via rich menu: `utm_source=rumors-line-bot&utm_medium=richmenu`
+  - If opened via push message: `utm_source=rumors-line-bot&utm_medium=push`
+  - If opened via LINE notify: `utm_source=line-notify&utm_medium=push`
```
Comment (nonumpa) [inline README.md:282]: According to [discussion](https://g0v.hackmd.io/OC7BneJ3TEGJHge1hWV-0w#Cofacts-Next-%E8%BF%BD%E8%B9%A4), we don't open articles page via LINE Notify anymore.

Comment (MrOrz) [reply README.md]: Nice catch! Thanks.

**Pull request merged**